### DISCUSSION TITLE
Can we access Shadow-root user-agent elements?

### DISCUSSION THREAD

**Discussion by DeepaGitWorld** [sukgu/shadow-automation-selenium#61] [General]
I have elements which are having shadow-root(user-agent). I went through some videos that all say that we cannot access shadow-root which is of type user-agent or closed. Can we access user-agent shadow dom using this library?

**Comment by sukgu** [comment]
Only open shadow root elements are available to access from outside. Please check this https://stackoverflow.com/a/63927668
